src/main/webapp/file-edit-button-fragment.xhtml

```diff
@@ -79,14 +79,14 @@
     </ui:fragment>
     
     <!-- Single file uningest/reingest -->
-    <ui:fragment rendered="#{isFilePg and (dataverseSession.user.superuser and FilePage.fileMetadata.dataFile.isTabularData()) or (FilePage.fileMetadata.dataFile.isIngestProblem() and FilePage.canPublishDataset())}">
+    <ui:fragment rendered="#{(dataverseSession.user.superuser and FilePage.fileMetadata.dataFile.isTabularData()) or (FilePage.fileMetadata.dataFile.isIngestProblem() and FilePage.canPublishDataset())}">
```

Comment:
Now I'm getting a 500 error when I try to go to the dataset page because the FilePage.fileMetadata is null (jakarta.servlet.ServletException: Cannot invoke "edu.harvard.iq.dataverse.FileMetadata.getDataFile()" because "this.fileMetadata" is null)

Reply:
Maybe it's not worth it to allow display the uningest on the dataset page. it will be pretty complicated - especially if the user selects multiple files from the list.

Reply:
OK - the list part shouldn't be an issue if this is only in the kebab menu, but yeak, more work than just removing the isFilePage check - I'll restore that.